integrationTests/testProcessorNode.go

```diff
@@ -412,11 +417,15 @@ func newBaseTestProcessorNode(args ArgTestProcessorNode) *TestProcessorNode {
 		nodesCoordinatorInstance = getDefaultNodesCoordinator(args.MaxShards, pksBytes)
 	}
 
-	peersRatingHandler, _ := p2pFactory.NewPeersRatingHandler(
-		p2pFactory.ArgPeersRatingHandler{
-			TopRatedCache: testscommon.NewCacherMock(),
-			BadRatedCache: testscommon.NewCacherMock(),
-		})
+	appStatusHandler := args.AppStatusHandler
+	if check.IfNil(args.AppStatusHandler) {
+		appStatusHandler = TestAppStatusHandler
```

Comment:
Is this OK? Having the same pointer TestAppStatusHandler will make debugging incredibly hard because nodes will tend to alter the same instance. Maybe not for this PR I will consider removing this TestAppStatusHandler and create different instances always.

Reply:
indeed, shouldn't use the same pointer